doc/topics/releases/3003.rst

```diff
@@ -23,3 +23,4 @@ Added
 -----
 
 - Firewall groups support to Vultr Salt Cloud provider
+- The ``salt-run`` CLI now accepts ``--jid`` which allows scripting against it.
```

Comment:
Changelog file is needed as well

Reply:
K. I'll create an issue for this since we need to link to one

Reply:
Done